File-Formats/Presentation/Presentation-to-PDF.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Converting PowerPoint Presentation to PDF |Syncfusion|
-description: This section illustrates how to convert PowerPoint Presentation document to PDF; PowerPoint Presentation conversion
+title: Converting Presentation to PDF in File Formats Presentation | Syncfusion
+description: This section illustrates how to convert PowerPoint Presentation document to PDF; PowerPoint Presentation conversion.
 platform: file-formats
 control: PowerPoint
 documentation: UG
 ---
-# Presentation to PDF conversion
+# Presentation to PDF conversion in Fileformat Presentation
```

Comment:
Change like this - Presentation to PDF conversion in File Formats Presentation

Reply:
Resolved